### PR TITLE
Fix Az.DataBoxEdge Syntax Errors

### PR DESCRIPTION
**Pull request by v-yuzhichen**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/DataBoxEdge/DataBoxEdge/help/New-AzDataBoxEdgeRole.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ The **New-AzDataBoxEdgeRole** cmdlet creates a new Role for a Data Box Edge devi
 
 ### Example 1
 ```powershell
-New-AzDataBoxEdgeRole -ResourceGroupName resourceGroupName -DeviceName deviceName -Name iotrole -DeviceProperties
+New-AzDataBoxEdgeRole -ResourceGroupName resourceGroupName -DeviceName deviceName -Name iotrole -DeviceProperty
  -IotDeviceId iotDeviceId -IotDeviceAccessKey iotAccessKey -IotEdgeDeviceId iotEdgeDeviceId -IotEdgeDeviceAccessKey iotEdgeDeviceAccessKey
```
Comment (ziyuezh576) [inline src/DataBoxEdge/DataBoxEdge/help/New-AzDataBoxEdgeRole.md:40]: add " `" at the end

File: src/DataBoxEdge/DataBoxEdge/help/New-AzDataBoxEdgeRole.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ The **New-AzDataBoxEdgeRole** cmdlet creates a new Role for a Data Box Edge devi
 
 ### Example 1
 ```powershell
-New-AzDataBoxEdgeRole -ResourceGroupName resourceGroupName -DeviceName deviceName -Name iotrole -DeviceProperties
+New-AzDataBoxEdgeRole -ResourceGroupName resourceGroupName -DeviceName deviceName -Name iotrole -DeviceProperty
```
Comment (ziyuezh576) [inline src/DataBoxEdge/DataBoxEdge/help/New-AzDataBoxEdgeRole.md:39]: add " `" at the end

File: src/DataBoxEdge/DataBoxEdge/help/New-AzDataBoxEdgeStorageAccountCredential.md [hunk 3]
```diff
@@ -25,7 +25,7 @@ The **New-AzDataBoxEdgeStorageAccountCredential** cmdlet creates a new edge stor
 
 ### Example 1
 ```powershell
-New-AzDataBoxEdgeStorageAccountCredential -ResourceGroupName resourceGroupName -DeviceName device-name -Name storage-acount-credential-name -StorageAccountName storageAccountName -StorageAccountType BlobStorage -StorageAccountAccessKey @SecureString -EncryptionKey @SecureString
+New-AzDataBoxEdgeStorageAccountCredential -ResourceGroupName resourceGroupName -DeviceName device-name -Name storage-acount-credential-name -StorageAccountType BlobStorage -StorageAccountAccessKey @SecureString -EncryptionKey @SecureString
 ```
```
Comment (ziyuezh576) [inline src/DataBoxEdge/DataBoxEdge/help/New-AzDataBoxEdgeStorageAccountCredential.md:29]: -Name storage-acount-credential-name -StorageAccountName storageAccountName
↓
-Name storageAccountName
(not sure)

File: src/DataBoxEdge/DataBoxEdge/help/Get-AzDataBoxEdgeDevice.md [hunk 4]
```diff
@@ -108,7 +108,7 @@ deviceName      resourceGroupName    Edge    eastus
 
 ### Example 3
 ```powershell
-Get-AzDataBoxEdgeDevice -ResourceGroupName resourceGroupName -DeviceName deviceName
+Get-AzDataBoxEdgeDevice -ResourceGroupName resourceGroupName -Name deviceName
```
Comment (ziyuezh576) [inline src/DataBoxEdge/DataBoxEdge/help/Get-AzDataBoxEdgeDevice.md:111]: edit L167

**Review by ziyuezh576 — APPROVED** [review]

**Review by BethanyZhou — APPROVED** [review]
Nice catch for the description of `Name`.

**Pull request merged**